### ISSUE TITLE
Allow Google scholar repository indexing: Conform to Meta tag guidelines

### ISSUE THREAD

**Issue by raprasad** [IQSS/dataverse#2445]
#### Follow the google scholar guidelines for searching repositories

Google scholar can natively search/crawl Eprints, DSpace, Digital Commons or OJS.  Dataverse contains the metadata to comply with these guidelines.

The guidelines are here:
- Go to: http://scholar.google.com/intl/en-us/scholar/inclusion.html#indexing
- Scroll to:  `2. Configuring the meta-tags`
- Excerpt:

> If you're using repository or journal management software, such as Eprints, DSpace, Digital Commons or OJS, please configure it to export bibliographic data in HTML "<meta>" tags. Google Scholar supports Highwire Press tags (e.g., citation_title), Eprints tags (e.g., eprints.title), BE Press tags (e.g., bepress_citation_title), and PRISM tags (e.g., prism.title). Use Dublin Core tags (e.g., DC.title) as a last resort - they work poorly for journal papers because Dublin Core doesn't have unambiguous fields for journal title, volume, issue, and page numbers. To check that these tags are present, visit several abstracts and view their HTML source. 

Sample code from the link above.  (We can easily generate this from metadata)

``` xml

<meta name="citation_title" content="The testis isoform of the phosphorylase kinase catalytic subunit (PhK-T) plays a critical role in regulation of glycogen mobilization in developing lung">
<meta name="citation_author" content="Liu, Li">
<meta name="citation_author" content="Rannels, Stephen R.">
<meta name="citation_author" content="Falconieri, Mary">
<meta name="citation_author" content="Phillips, Karen S.">
<meta name="citation_author" content="Wolpert, Ellen B.">
<meta name="citation_author" content="Weaver, Timothy E.">
<meta name="citation_publication_date" content="1996/05/17">
<meta name="citation_journal_title" content="Journal of Biological Chemistry">
<meta name="citation_volume" content="271">
<meta name="citation_issue" content="20">
<meta name="citation_firstpage" content="11761">
<meta name="citation_lastpage" content="11766">
<meta name="citation_pdf_url" content="http://www.example.com/content/271/20/11761.full.pdf">
```

**Comment by bencomp** [comment]
Dupe of #1393, I think.

**Comment by jggautier** [comment]
Closing this issue, which was resolved in a slightly different way in #1393.
